specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/logic.json

```diff
@@ -6995,7 +7221,8 @@
         "NotSpecified",
         "AS2",
         "X12",
-        "Edifact"
+        "Edifact",
+        "RosettaNet"
```

Comment:
>RosettaNet [](start = 9, length = 10)

You're adding an enum value to an enum that is not modeled as string in a stable API version. This is considered a breaking change because an existing client will not be able to deserialize a GET response. You need to introduce a new API version for this.

Reply:
Thanks for removing the breaking changes.

---
In reply to: [308998551](https://github.com/Azure/azure-rest-api-specs/pull/6702#discussion_r308998551) [](ancestors = 308998551)